tests/gnrc_netif/Makefile

```diff
@@ -21,6 +21,10 @@ USEMODULE += netdev_ieee802154
 USEMODULE += netdev_test
 USEMODULE += od
 
+# deactivate automatically emitted packets from IPv6 neighbor discovery
+CFLAGS += -DGNRC_IPV6_NIB_CONF_ARSM=1
```

Comment:
looks unrelated, at least not used above

Reply:
the address state machine causes sending of unsolicited neighbor solicitations.

Reply:
Arghs, but it shoud be 0. You are right!